app/views/projects/show/_complete.html.erb

```diff
@@ -1,5 +1,6 @@
-<div class="govuk-grid-row govuk-body">
-  <div class="govuk-grid-column-two-thirds">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-quarter">&nbsp;</div>
```

Comment:
Gah! Is this what we have to do? Could we put the side nav and main content into a row instead?